### PR TITLE
init script with yq

### PR DESCRIPTION
**Pull request by arya-harness**

### CODE REVIEW

File: src/harness/scripts/migrate-0.9.x.sh [hunk 1]
```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:1]: NIT: consider renaming the file to be more descriptive. migrate-values-0.9.x, restructure-values-0.9.x?

File: src/harness/scripts/migrate-0.9.x.sh [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      ;;
+    \? )
+      echo "Invalid option: $OPTARG" 1>&2
+      echo "use ./migrate-0.9.x.sh -f <override.yaml>  "
+      ;;
+    : )
+      echo "Invalid option: $OPTARG requires an argument" 1>&2
+      ;;
+  esac
+done
```
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:18]: Validate and exit early if arguments/flags are not provided/invalid

File: src/harness/scripts/migrate-0.9.x.sh [hunk 3]
```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+# pass 2 file names to this script, it will yield a output
```
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:2]: Consider exiting early if an error/validation fails

File: src/harness/scripts/migrate-0.9.x.sh [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+# - mongo moved from platform.mongo to platform.boostrap.database.mongo
+yq eval '.platform.bootstrap.database.mongodb = .platform.mongodb | del(.platform.mongodb)' -i "$newOverrideFile"
+echo "Moved platform.mongodb to platform.bootstrap.database.mongodb in $newOverrideFile"
+# - timescaledb moved from platform.timescaledb to platform.boostrap.database.timescaledb
+yq eval '.platform.bootstrap.database.timescaledb = .platform.timescaledb | del(.platform.timescaledb)' -i "$newOverrideFile"
+echo "Moved platform.timescaledb to platform.bootstrap.database.timescaledb in $newOverrideFile"
+# - minio moved from platform.minio to platform.boostrap.database.minio
+yq eval '.platform.bootstrap.database.minio = .platform.minio | del(.platform.minio)' -i "$newOverrideFile"
+echo "Moved platform.minio to platform.bootstrap.database.minio in $newOverrideFile"
+# - minio moved from platform.redis to platform.boostrap.database.redis
```
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:65]: type: should be redis instead of minio.

File: src/harness/scripts/migrate-0.9.x.sh (hunk 1 above)
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:1]: Add the following:

Add the following:

set -o errexit  
set -o pipefail 
set -o nounset

File: src/harness/scripts/migrate-0.9.x.sh [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+if command -v yq &>/dev/null; then
+    echo "yq is installed."
+else
+    echo " yq is not installed."
+    echo "\n For linux system, Run the following: \n ------------------- \n wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq \n ------------------- "
+    echo "\n For Windows system, Run the following: \n ------------------- \n choco install yq \n ------------------- "
+    echo "\n For other system and information, visit : \n ------------------- \n https://github.com/mikefarah/yq \n ------------------- "
+fi
+old_file=$overrideFile
+new_file_name="newOverride.yaml"
```
Comment (saitejavn-harness) [inline src/harness/scripts/migrate-0.9.x.sh:30]: consider new file name to be based on input file name.

Comment (arya-harness) [reply src/harness/scripts/migrate-0.9.x.sh:30]: done

File: src/harness/scripts/migrate-0.9.x.sh (hunk 1 above)
Comment (arya-harness) [reply src/harness/scripts/migrate-0.9.x.sh:1]: done

File: src/harness/scripts/migrate-0.9.x.sh (hunk 4 above)
Comment (arya-harness) [reply src/harness/scripts/migrate-0.9.x.sh:65]: done

File: src/harness/scripts/migrate-0.9.x.sh (hunk 3 above)
Comment (arya-harness) [reply src/harness/scripts/migrate-0.9.x.sh:2]: done

File: src/harness/scripts/migrate-0.9.x.sh (hunk 1 above)
Comment (arya-harness) [reply src/harness/scripts/migrate-0.9.x.sh]: done

**Pull request merged**